pkg/scd/store/cockroach/subscriptions.go

```diff
@@ -185,15 +185,42 @@ func (c *repo) pushSubscription(ctx context.Context, q dsssql.Queryable, s *scdm
 			FROM
 				scd_subscriptions
 			WHERE
-				id = $1
+				id = $1::uuid
 		)
-		UPSERT INTO
+		INSERT INTO
 		  scd_subscriptions
 		  (%s)
 		VALUES
 			($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, transaction_timestamp())
+		ON CONFLICT (%s) DO UPDATE
+			SET %s = $2,
+				%s = $3::int,
+				%s = $4,
+				%s = $5::int,
+				%s = $6,
+				%s = $7,
+				%s = $8::bool,
```

Comment:
Is setting the types here required for yugabyte? If so, why is it not required as well for operational intents?